Makefile

```diff
@@ -0,0 +1,7 @@
+marianpci-y := marian.o a3.o generic.o m2.o seraph8.o spi.o
+obj-m := marianpci.o
+
```

Comment:
What sets KSRC?

"Traditionally" you would provide this in the makefile with a lines like:
```
KERNELVER       ?= $(shell uname -r)
KERNELDIR       ?= /lib/modules/$(KERNELVER)/build
```